circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/Replica.java

```diff
@@ -89,6 +93,29 @@
     this.housekeepingListener = housekeepingListener;
     replicationMode = tableReplication.getReplicationMode();
     this.tableReplication = tableReplication;
+    this.partitionBatchSize = DEFAULT_PARTITION_BATCH_SIZE;
+  }
+
+  /**
+   * Use {@link ReplicaFactory}
```

Comment:
What does this comment mean?

Reply:
It was copied from the other constructor, and it means that this constructor is package-private and to construct this, you need to use `ReplicaFactory`.

Reply:
Ah, OK.